tests/libdv/dv.go

```diff
@@ -68,19 +68,18 @@ func WithName(name string) dvOption {
 	}
 }
 
-type pvcOption func(*corev1.PersistentVolumeClaimSpec)
+type storageOption func(*v1beta1.StorageSpec)
 
-// WithPVC is a dvOption to add a PVCOption spec to the DataVolume
-// The function receives an optional list of pvcOption, to override the defaults
+// WithStorage is a dvOption to add a StorageOption spec to the DataVolume
```

Comment:
Should we fully deprecate `WithPVC`?

Reply:
I can see cases where it might still be useful. But I can fully remove it if needed.

Reply:
Yeah I'd prefer if we kept it, just in case

Reply:
Well found a case that needed it so I restored it and the pieces of functionality it needed.